src/Service/ActivityPubManager.php

```diff
@@ -173,6 +172,11 @@ public function findActorOrCreate(?string $actorUrlOrHandle): User|Magazine|null
             return $this->userRepository->findOneBy(['username' => $name]);
         }
 
+        // Check if the instance is banned
+        if ($this->settingsManager->isBannedInstance($actorUrl)) {
+            return null;
+        }
```

Comment:
I think we should just throw a `InstanceBannedException`. This would make it behave the same way it did before. Most of the code would probably work fine if we returned `null`, but it would be a big change in the behavior...

Reply:
See my second point: I added additional checks for these methods in ActivityPubManager now as well. And just return null, which just ignore the request. Without the need for additional try/catch blabla in the callee methods.

Reply:
.. This is because `findActorOrCreate` is called outside of this file/class. Which doesn't have all the try/catch in place, making it therefor unnecessary complex to call this method.